src/deadline/keyshot_adaptor/KeyShotClient/keyshot_client.py

```diff
@@ -21,6 +21,8 @@
                 print("add_dll_directory failed: %s" % p)
         sys.path.append(p)
 
+# Required for making pywin32 portable
+import pywin32_bootstrap  # type: ignore # noqa: F401 E402
```

Comment:
Any link or paper trail for why this is necessary?

Reply:
Good call. There's some info here: https://github.com/mhammond/pywin32/blob/main/win32/Lib/pywin32_bootstrap.py#L1-L8

Usually it gets run automatically as the module is loaded by a `.pth`, but `.pth` files are only run when modules are in the site-packages folder from what I've read.